RELEASE_NOTES.md

```diff
@@ -28,4 +28,13 @@ Authentication Service MKII release notes
 * the `POST /logout` endpoint now returns JSON if requested and also deletes any temporary link
   tokens associated with the user from the database. Any temporary link or temporary login
   cookies are removed. If JSON is requested, the login cookie is not removed, unlike with an
-  HTML response.
\ No newline at end of file
+  HTML response.
+
+0.2.1
+-----
+
+* Added dockerfile that is compatible with automated docker build practices.
+* Docker image is built against alpine JRE for a smaller footprint
```

Comment:
Is this still true? I thought you changed the base image...?

Reply:
Yeah, out of date and I've fixed it.